src/anyon/anyon.jl

```diff
@@ -282,6 +282,14 @@ function get_metadata(client::Client, qpu::UnionAnyonQPU)::Metadata
         output["excluded_positions"] = Vector{Int}()
     end
 
+    if haskey(machineMetadata, "excludedCouplers")
+        raw_excluded_couplers =
+            convert(Vector{Vector{Int}}, machineMetadata["excludedCouplers"])
+        output["excluded_couplers"] = convert_excluded_couplers(raw_excluded_couplers)
+    else
+        output["excluded_couplers"] = Vector{Tuple{Int,Int}}()
```

Comment:
To be more general, we can make this "excluded_connections", this will also be more consistent with the general vocabulary of "excluded_positions"